drape_frontend/apply_feature_functors.cpp

```diff
@@ -775,12 +771,12 @@ void ApplyAreaFeature::ProcessRule(AreaRuleProto const & areaRule, double areaDe
 ApplyLineFeatureGeometry::ApplyLineFeatureGeometry(TileKey const & tileKey, TInsertShapeFn const & insertShape,
                                                    FeatureType & f, double currentScaleGtoP)
   : TBase(tileKey, insertShape, f, CaptionDescription())
+  , m_spline(f.GetPointsCount())
   , m_currentScaleGtoP(currentScaleGtoP)
   // TODO(pastk) : calculate just once in the RuleDrawer.
-  , m_minSegmentSqrLength(base::Pow2(4.0 * df::VisualParams::Instance().GetVisualScale() / currentScaleGtoP))
   , m_simplify(tileKey.m_zoomLevel >= 10 && tileKey.m_zoomLevel <= 12)
+  , m_minSegSqLength(m_simplify ? base::Pow2(4.0 * df::VisualParams::Instance().GetVisualScale() / currentScaleGtoP) : 0)
```

Comment:
What is this formula about? A comment would be helpful.

Reply:
Self-explained here, 4px (with vs) to global coord.